README.md

```diff
@@ -6,8 +6,8 @@ What is Notepad++ ?
 
 Notepad++ is a free (free as in both "free speech" and "free beer") source code
 editor and Notepad replacement that supports several programming languages and
-natural languages. Running in the MS Windows environment, its use is governed by
-GPL License.
+natural languages. Running in the MS-Windows environment, its use is governed by

```

Comment:
This is actually introducing an error. The correct form is "Microsoft Windows", or "MS Windows" for short. MS-Windows suggests a variant of some more generic Windows product that was specifically built by MS.